docs/content/en/docs/pipeline-stages/deployers/helm.md

```diff
@@ -18,9 +18,179 @@ To use `helm` with Skaffold, the `helm` binary must be installed on your machine
 
 Skaffold supports projects set up to deploy with Helm, but certain aspects of the project need to be configured correctly in order for Skaffold to work properly. This guide should demystify some of the nuance around using Skaffold with Helm to help you get started quickly.
 
-{{< alert title="No more `artifactsOverride`" >}}
-Skaffold no longer requires the intricate configuring of `artifactsOverride` and image naming strategies.
-{{< /alert >}}
+## Image Configuration
```

Comment:
Thanks! Just confirming these are the v1 docs right?

Reply:
Yes! Is the same version that is currently on [production](https://skaffold.dev/docs/pipeline-stages/deployers/helm/), plus the change to improve the consistency using the `image` key. The preview URL is already generated in case you want to double check: http://35.235.77.76:1313/